### DISCUSSION TITLE
problem with Chia software

### DISCUSSION THREAD

**Discussion by shahrooznik** [Chia-Network/chia-blockchain#8698] [Support]
Hello
I have a problem connecting Chia software
How do I fix the following error?
"
ERROR    Failed to update farmer information on the pool https://eu1.pool.space
"

I am using Windows 10 and I have no problem with the previous two computers where I installed the bag.
Please help me how to fix this error?

Regards,
Shahrooz

**Comment by DPRC137** [comment]
![Screenshot 2021-10-04 162540](https://user-images.githubusercontent.com/50619554/135839580-f92b4555-e4f3-4002-9414-fb120302884e.png)

Link to change payout address: https://blog.pool.space/how-to-change-your-payout-address-in-space-pool-df7441f0bfdc

**Comment by BrandtH22** [comment]
@shahrooznik , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
